jobsdb/jobsdb.go

```diff
@@ -208,6 +208,7 @@ func loadConfig() {
 	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
 	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
 	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+	sslmode = config.GetEnv("JOBS_DB_SSLMODE", "disable")
```

Comment:
Thanks for the pull request.



https://godoc.org/github.com/lib/pq

We need to check if that's a valid mode as mentioned in the above docs, if not fallback to disable.

Reply:
Right, will check from the list of valid ssl modes given in `pq`'s documentation.